app/controllers/admin/positions_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Admin::PositionsController < ApplicationController
+  before_action -> { user_has_position(Position.first) }
```

Comment:
even if you're using user_has_position the way you already wrote it grabbing the first one still seems dicey cause there is no guarantee as to which Position first is going to return right?